[翻译完成] flutter.cn/src/ads/index.md (#1116)
cfug/flutter.cn: src/ads/index.md

以下的视频教程，[通过 Flutter 应用获利][Monetizing apps with Flutter]，展示了如何开始使用广告功能。

<iframe width="560" height="315" src="https://player.bilibili.com/player.html?aid=289460171&bvid=BV1Vf4y147Er&cid=305747760&page=1" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe><br>

The following resources can help you get started:

以下资源可以帮助你入门：

* To get the Google Mobile Ads SDK for Flutter,
  download the [google_mobile_ads plugin][plugin] from pub.dev.
  
  要获取 Flutter 的谷歌移动广告 SDK ，请在 pub.dev 下载 [google_mobile_ads plugin][plugin] 插件。

* For instructions on creating and loading an Ad with
  AdMob or AdManager, see the [detailed implementation guide][].

  对于如何使用 AdMob 或 AdManager 创建和加载广告的相关说明，可以查阅 [detailed implementation guide][] 文档。

* To learn how to implement overlay banner,
  interstitial, and rewarded ads, see the
  [Adding AdMob ads to a Flutter app][] codelab.

  想要了解如何实现叠加横幅广告、插页式广告和奖励广告，可以查阅 [Adding AdMob ads to a Flutter app][] 文档。

* To learn how to implement inline banner
  and native ads, see the [Adding AdMob banner
  and native inline ads to a Flutter app][] codelab.

>   想要了解如何实现内嵌横幅广告和本地广告，可以查阅 [Adding AdMob banner
>   and native inline ads to a Flutter app][] 文档。

Reviewer: ```suggestion
  想要了解如何实现内嵌横幅广告和本地广告，可以查阅
  [添加 AdMob 的横幅和信息流广告到 Flutter 应用中][
  Adding AdMob banner and native inline ads to a Flutter app][]
  的代码实验室。
```

* If you experience any problems with the beta release,
  please [file an issue][].

>   如果你在测试版本中遇到任何问题，请 [提出问题][file an issue] 。

Reviewer: ```suggestion
  如果你在测试版本中的尝试中遇到任何问题，请向插件仓库 [提出反馈][file an issue] 。
```

interstitial, rewarded video, native ads,
and adaptive banner.

你可以借助 [Google Flutter 版移动广告 SDK][plugin] 在项目中集成广告功能，

> 这款插件可以与 AdMob 和 AdManager 一起使用。
> 这款插件还支持多种广告格式，包括横幅广告（内嵌和叠加）、插页式广告、奖励视频广告、原生广告和自适应横幅广告。

Reviewer 2: ```suggestion
这款插件可以与 AdMob 和 AdManager 一起使用。
支持多种广告格式，包括横幅广告（内嵌和叠加）、插页式广告、奖励视频广告、原生广告和自适应横幅广告。
```

![Pic showing different types of ads]({{site.url}}/assets/images/ads/GoogleMobileAdTypes.png){:width="100%"}

The following video tutorial,
[Monetizing apps with Flutter][],
shows how to get started with Ads:

以下的[通过 Flutter 应用获利][Monetizing apps with Flutter]
视频教程展示了如何开始使用广告功能:

<iframe width="560" height="315" src="https://player.bilibili.com/player.html?aid=289460171&bvid=BV1Vf4y147Er&cid=305747760&page=1" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe><br>

The following resources can help you get started:

以下资源可以帮助你入门：

* To get the Google Mobile Ads SDK for Flutter,
  download the [google_mobile_ads plugin][plugin] from pub.dev.

  在 pub.dev 下载 [google_mobile_ads 插件][plugin] 以获取 Flutter 的 Google 移动广告 SDK。

* For instructions on creating and loading an Ad with
  AdMob or AdManager, see the [detailed implementation guide][].

  有关如何使用 AdMob 或 AdManager 创建和加载广告的相关说明，可以查阅
  [详细实现的指导][detailed implementation guide] 文档。

* To learn how to implement overlay banner,
  interstitial, and rewarded ads, see the
  [Adding AdMob ads to a Flutter app][] codelab.

  想要了解如何实现叠加横幅广告、插页式广告和奖励广告，可以查阅

>   [添加 AdMob 到 Flutter 应用中][Adding AdMob ads to a Flutter app] 的代码实验室。

Reviewer 2: ```suggestion
  [添加 AdMob 到 Flutter 应用中][Adding AdMob ads to a Flutter app] 的 Codelabs 教程。
```
和下面统一下

* To learn how to implement inline banner
  and native ads, see the [Adding AdMob banner
  and native inline ads to a Flutter app][] codelab.

  想要了解如何实现内嵌横幅广告和本地广告，可以查阅
  [添加 AdMob 的横幅和信息流广告到 Flutter 应用中][Adding AdMob banner and native inline ads to a Flutter app]
  的 Codelabs 教程。

* If you experience any problems with the beta release,
  please [file an issue][].

>   如果你在测试版本中的尝试中遇到任何问题，请向插件仓库 [提出反馈][file an issue] 。

Reviewer 2: ```suggestion
  如果你在公开试用版本遇到了任何问题，请 [向我们反馈][file an issue] 。
```
